2018-09-05 11:18:47,397 - DataBase.py[line:23] - INFO: database class is creating...
2018-09-05 11:18:47,467 - DataBase.py[line:51] - INFO: database is connecting...
2018-09-05 11:19:00,546 - DataBase.py[line:56] - INFO: select * from empinfo where Emp_name=name1
2018-09-05 11:19:00,611 - DataBase.py[line:62] - ERROR: query database exception, (1054, "Unknown column 'name1' in 'where clause'")
2018-09-05 11:19:39,783 - DataBase.py[line:56] - INFO: select * from empinfo where Emp_id=000001
2018-09-05 11:20:15,471 - DataBase.py[line:56] - INFO: select * from empinfo where Emp_name=name1
2018-09-05 11:20:15,526 - DataBase.py[line:62] - ERROR: query database exception, (1054, "Unknown column 'name1' in 'where clause'")
